src/app/services/guess-word.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class GuessWordService {
+  constructor(private http: HttpClient) {}
+
+  baseURL: string = '';
```

Comment:
```suggestion
  baseURL = '';
```
Don't declare types if there's a assignation who can infer the type already